pkg/k8sutil/portforward.go

```diff
@@ -86,26 +86,30 @@ func PortForward(kubeContext string, localPort int, remotePort int, namespace st
 
 	forwarder, err := portforward.New(dialer, []string{fmt.Sprintf("%d:%d", localPort, remotePort)}, stopChan, readyChan, out, errOut)
 	if err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
+	errChan := make(chan error, 2) // 2 go routines are writing to this channel
+
 	go func() {
 		for range readyChan {
```

Comment:
this seems funny... why isnt it just?

```go
<-readyChan
```

Reply:
this was written through trial and error... that channel is constantly written to, which eventually blocks, and it's nothing we care about